context/sm_context.go

```diff
@@ -39,6 +39,8 @@ type SmContext struct {
 	// for duplicate pdu session id handling
 	UlNASTransportVal *nasMessage.ULNASTransport
 	DuplicatedVal     bool
+
+	SmfProfiles []models.NfProfile
```

Comment:
Temp solution. We need to use NRF cache lib code.

Reply:
Yes, this is temporary fix until NRF cache lib codes is not available